blogs/2016/10/31/js_roundup_2.md

```diff
@@ -0,0 +1,133 @@
+---
+Order: 18
+TOCTitle: JavaScript Extensions Part 2
+PageTitle: Visual Studio Code JavaScript Extensions Part 2 Oct 2016
+MetaDescription: Essential JavaScript extensions for Visual Studio Code. 
+Date: 2016-10-31
+ShortDescription: Essential JavaScript extensions for Visual Studio Code. 
+Author: Wade Anderson
+---
+
+# JavaScript Extensions Part 2
+
+October 31, 2016 by Wade Anderson, [@waderyan_](https://twitter.com/waderyan_)
+
+Visual Studio Code has excellent support for JavaScript out of the box. As with other languages, JavaScript is powered by a language service. The [JavaScript language service](https://github.com/Microsoft/TypeScript/wiki/Salsa) is implemented by the TypeScript team, allowing JavaScript developers to leverage the best IntelliSense experience. 
+
+But what other features can you get with VS Code? Because of VS Code's rich extensibilty model, there are many features extensions can provide. This post is a follow up to my previous post about JavaScript extensions and a brief introduction to writing your own JavaScript extension. 
```

Comment:
VS Code has a rich extensibility model and many features are provided through VS Code extensions.